### ISSUE TITLE
Callback Task

### ISSUE THREAD

**Issue by Taytan9912** [zzm317/MyOwnWorld#1]
This plugin attempted to register a deprecated callback task, so how i can use/set this ? Please help

**Comment by zzm317** [comment]
=== I just too lazy to change the task
The old callback task can be used
Just a warning why you get stuck on it==
Just ignore it

**Comment by Taytan9912** [comment]
Ok, i have second question, whether mobs respawning randomly on the world ?

**Comment by zzm317** [comment]
yes,but not all the map,mobs spawn in a circle where players are.
